### ISSUE TITLE
Anybody observe slow training speed for mamba compared to transformer model?

### ISSUE THREAD

**Issue by vgthengane** [LMD0311/PointMamba#14]

**Comment by d62lu** [comment]
yes, I just made the comparison. The speed of the Mamba block is truly slower than Transformer, under the same input dimension

**Comment by d62lu** [comment]
for both model training and inference. I am not looking into the details of the Mamba block, maybe I missed something in the code....
